### PR TITLE
{math}[foss/2022b] pauvre v0.2.3

### PR DESCRIPTION
**Pull request by jkuhl-uni**
Needed for #20340

### CODE REVIEW

File: easybuild/easyconfigs/p/pauvre/pauvre-0.2.3-foss-2022b.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+
```
[the remark this answers is not in the document]
Comment (jkuhl-uni) [reply easybuild/easyconfigs/p/pauvre/pauvre-0.2.3-foss-2022b.eb:1]: Sure, sorry for that

**Comment by jkuhl-uni** [comment]
Hey @verdurin,
can you maybe have a look at this again, so we can also merge #20340 ?

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/aca9af7969fabe594f1a454c20167af9 for a full test report.

File: easybuild/easyconfigs/p/pauvre/pauvre-0.2.3-foss-2022b.eb (hunk 1 above)
[the remark this answers is not in the document]
Comment (verdurin) [reply easybuild/easyconfigs/p/pauvre/pauvre-0.2.3-foss-2022b.eb:3]: Thanks.

**Review by verdurin — APPROVED** [review]

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen3

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
interactive2 - Linux Rocky Linux 8.9, x86_64, AMD EPYC-Milan Processor (zen2), Python 3.6.8
See https://gist.github.com/bedroge/39b0ce93beb03dfa849f21a972d849a3 for a full test report.

**Review by bedroge — APPROVED** [review]
Looks good to me.

**Comment by bedroge** [comment]
Going in, thanks @jkuhl-uni!

**Pull request merged**